l1-contracts/src/core/Rollup.sol

```diff
@@ -49,14 +49,26 @@ contract Rollup is EIP712("Aztec Rollup", "1"), Leonidas, IRollup, ITestRollup {
   struct BlockLog {
     bytes32 archive;
     bytes32 blockHash;
+    // TODO(Miranda): Is this the best place to store some blob info?
+    bytes32 blobHash;
     Slot slotNumber;
   }
 
+  // Blob public inputs are stored when we publish blocks to link DA to a L1 blob. They are read and used
+  // when verifying an epoch proof to link DA to our L2 blocks.
+  struct BlobPublicInputs {
+    bytes32 z;
+    bytes32 y;
+    bytes32[2] c;
+  }
+
   // See https://github.com/AztecProtocol/engineering-designs/blob/main/in-progress/8401-proof-timeliness/proof-timeliness.ipynb
   // for justification of CLAIM_DURATION_IN_L2_SLOTS.
   uint256 public constant CLAIM_DURATION_IN_L2_SLOTS =
     Constants.AZTEC_EPOCH_PROOF_CLAIM_WINDOW_IN_L2_SLOTS;
   uint256 public constant PROOF_COMMITMENT_MIN_BOND_AMOUNT_IN_TST = 1000;
+  // Always true, exists to override to false for testing only
+  bool public checkBlob = true;
```

Comment:
Don't put mutable values before immutable values. Linter will likely scream at you, if not we seem to be missing a rule.

Reply:
Will do - no screaming is happening btw

Reply:
Locally or in ci, could be solhint versioning issues